beacon-chain/rpc/service.go

```diff
@@ -24,29 +24,22 @@ import (
 
 var log = logrus.WithField("prefix", "rpc")
 
-// canonicalFetcher defines a struct with methods that can be
-// called on-demand to fetch the latest canonical head
-// and crystallized state as well as methods that stream
-// latest canonical head events to clients
-// These functions are called by a validator client upon
-// establishing an initial connection to a beacon node via gRPC.
-type canonicalFetcher interface {
+type beaconDB interface {
```

Comment:
Could you name this to something else ? The name gives the impression that this interface can be used to perform the normal db operations  get,put,has,delete,etc

Reply:
nvm ignore this point, I just looked at the code again